specifyweb/frontend/js_src/lib/components/Syncer/syncers.ts

```diff
@@ -238,12 +238,23 @@ export const syncers = {
         const currentChildren =
           children[tagName] ?? children[tagName.toLowerCase()] ?? [];
         const child = currentChildren[0];
+
+        let hasCondition = false;
+        currentChildren.forEach((child) => {
+          if (
+            child.attributes.condition !== undefined &&
+            child.attributes.condition.length > 0
+          ) {
+            hasCondition = true;
+          }
+        });
```

Comment:
refactor to use .some
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

please look over js array methods and apply them more - they will make you feel more powerful!